build-system/tasks/process-github-issues.js

```diff
@@ -102,13 +111,12 @@ function getIssues(opt_page) {
  */
 function updateGitHubIssues() {
   let promise = Promise.resolve();
-  return BBPromise.all([
-    getIssues(1),
-    getIssues(2),
-    getIssues(3),
-    getIssues(4),
-    getIssues(5),
-  ])
+  const arrayPromises = [];
+  // we need to pull issues in batches
+  for (let batch = 1; batch < 11; batch++) {
```

Comment:
You're still hard coding the number of batches. Move it to a const `NUM_BATCHES` at the top of the file.

Reply:
This is not what I meant. You can simply make`NUM_BATCHES` a constant equal to 11 and use it in the loop you'd written earlier.

Reply:
oh, ok. Now I know what you mean. Done. Thank you.